src/bvals/comms/bnd_info.cpp

```diff
@@ -82,25 +82,30 @@ Indexer6D CalcLoadIndices(const NeighborIndexes &ni, TopologicalElement el,
                  // offset in some direction
     } else if (block_offset[dir] > 0) {
       s[dir] = bounds[dir].e - Globals::nghost + 1 - top_offset[dir];
-      e[dir] = bounds[dir].e - top_offset[dir];
+      e[dir] = bounds[dir].e;
     } else {
-      s[dir] = bounds[dir].s + top_offset[dir];
+      s[dir] = bounds[dir].s;
```

Comment:
Is the offset not required any more because that the topological element at at index is not owned?